lib/schema_plus/core/sql_struct.rb

```diff
@@ -12,6 +16,7 @@ def parse!(sql)
             \( \s*
             (?<body>.*) \s*
             \) \s*
+            #{INHERITANCE_REGEX if sql[INHERITANCE_KEY]}
```

Comment:
Unfortunately that wouldn't work (
Ruby uses the greedy algorithm in regex
and if `INHERITS` is not contained directly in regex then bug is reproduced and `INHERTIS` section will included into `body`
If 'greed' switch off for body - internal brackets in body will break all =(